manifests/g/GenericMappingTools/gmt/6.2.0/GenericMappingTools.gmt.yaml

```diff
@@ -0,0 +1,18 @@
+PackageIdentifier: GenericMappingTools.gmt
+Publisher: Generic Mapping Tools
+PackageName: Generic Mapping Tools
```

Comment:
```suggestion
PackageName: GMT6
```

Reply:
I actually prefer to name it "GMT", rather then "GMT6". Is it OK?

Reply:
If you want other people to upgrade the application on their device, it would have to be `GMT6` otherwise they may not be able to upgrade via winget upgrade, because both winget list and winget upgrade rely on the data shown in Control Panel's Add or Remove Programs.

Reply:
I see. Thanks for the explanation.